code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -225,7 +225,6 @@
 	reagent_state = SOLID
 	color = "#FFFFFF"
 	overdose = REAGENTS_OVERDOSE
-	taste_tag = list(TASTE_SALTY)
```

Comment:
return this label, and add below this line this:
sanity_gain_ingest = 0

Reply:
also add
sanity_gain_ingest = 0 to all the condiments and things we don't want to provide "rest".